### DISCUSSION TITLE
[Projects Beta] Enum 25 values limit

### DISCUSSION THREAD

**Discussion by Totktonada** [community/community#6525] [Projects and Issues] [answered]
We categorize backlog issues using a enum column: epics, future and in-progress tracks. I just observed that a enum can contain only 25 values and it is hard below amount of tracks we want to work on. To be honest, I don't know to how work it around. Maybe a text field and a lot of copy-pasting...

I would use something like https://github.com/github/feedback/discussions/5536 if there would be such functionality.

**Comment by ynouri** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/6419

**Reply by Totktonada** [reply]
Thanks, missed it.
